src/components/DynamicFormContainer/DynamicFormHeader/DynamicFormHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from "react";
+import { Button } from "../../../UI/Button";
+import { SvgIcon, SvgIconArrowLeft } from "../../../UI/SvgIcon";
+import { Title } from "../../../UI/Title";
+import { ProgressBar } from "../../ProgressBar";
+
+interface DynamicFormHeaderProps {
+  onBackToFormSelection: () => void;
+  onNewForm: () => void;
+  onFormSubmit: () => void;
+}
+
+export const DynamicFormHeader: FunctionComponent<DynamicFormHeaderProps> = ({
+  onBackToFormSelection,
+  onNewForm,
+  onFormSubmit,
+}) => {
+  return (
+    <>
```

Comment:
Can rm

Reply:
You also like to put fragments everywhere lol

Reply:
hahah ok